### DISCUSSION TITLE
2005/12/30/%E5%96%9C%E6%AC%A2%E7%8E%A9%E7%A5%96%E7%8E%9B%EF%BC%88zuma%EF%BC%89%E7%9A%84%E6%9D%A5%E8%AF%95%E8%AF%95%E8%BF%99%E4%B8%AA%E6%B8%B8%E6%88%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#72] [Announcements]
# 2005/12/30/%E5%96%9C%E6%AC%A2%E7%8E%A9%E7%A5%96%E7%8E%9B%EF%BC%88zuma%EF%BC%89%E7%9A%84%E6%9D%A5%E8%AF%95%E8%AF%95%E8%BF%99%E4%B8%AA%E6%B8%B8%E6%88%8F

我才发现的新游戏：《Giza》，是祖玛后续版本，与祖玛玩法类似，中文叫埃及祖玛。

https://blog.qizhen.xyz/2005/12/30/%E5%96%9C%E6%AC%A2%E7%8E%A9%E7%A5%96%E7%8E%9B%EF%BC%88zuma%EF%BC%89%E7%9A%84%E6%9D%A5%E8%AF%95%E8%AF%95%E8%BF%99%E4%B8%AA%E6%B8%B8%E6%88%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Unknown
2006年01月1日 11:36 编辑
今天第一次去了浦东机场，感觉很坏的说。看到几乎所有的人都带着无比笨重的大箱子，一想到还要转机取行李，不累死也会半死 :( 太恐怖了！我现在觉得出国是一个很痛苦的过程。

回复
幽
2005年12月31日 23:48 编辑
元旦快乐！！！！！！ ：）

回复
蟲蟲の蟲ル
2005年12月31日 22:35 编辑
对了，住你永远不牙疼，哈哈~

回复
蟲蟲の蟲ル
2005年12月31日 22:35 编辑
哈，第一次玩祖玛还是在大1的课堂上哦，呵呵~新年快乐，http://img003.photo.wangyou.com/2005/9/2/146606/200511360371030.jpg送上贺图：）
